packages/synapse-react-client/package.json

```diff
@@ -147,8 +147,8 @@
     "@types/lodash-es": "4.17.3",
     "@types/markdown-it": "^12.0.1",
     "@types/node": "^14.14.31",
-    "@types/plotly.js": "^1.54.20",
-    "@types/plotly.js-basic-dist": "^1.54.0",
+    "@types/plotly.js": "^2.12.18",
+    "@types/plotly.js-basic-dist": "^1.54.1",
```

Comment:
@types/plotly.js-basic-dist just mirrors the types in @types/plotly.js, so it's fine that it's at v1